### DISCUSSION TITLE
[Projects Beta] Field that links to other Issue(s) / Pull Request(s)

### DISCUSSION THREAD

**Discussion by Alexander01998** [community/community#14432] [Projects and Issues]
It is currently possible to achieve this with a text field containing a link to the issue/PR.
However, that gets really messy with multiple issues/PRs because each link is so long.

![image](https://user-images.githubusercontent.com/10100202/162055402-4160d83d-1680-46d7-82e0-a1722e6ffe3a.png)

![image](https://user-images.githubusercontent.com/10100202/162057617-aa1bf19b-f3fe-48df-8827-1d22ea0ba7be.png)

**Comment by strasdat** [comment]
I'd like this too. However, since there can be multiple repositories in a project, it should be something like #REPO_NAME/511 or similar.
